docs/pages/machine-id/access-guides/databases.mdx

```diff
@@ -140,114 +104,129 @@ supported `format` options is below:
 | CockroachDB   | `cockroach`   | Provides `cockroach/node.key`, `cockroach/node.crt`, and `cockroach/ca.crt`. |
 | Generic TLS   | `tls`         | Provides `tls.key`, `tls.crt`, and `tls.cas` for generic clients that require specific file extensions. |
 
-Be sure to configure the `token` and `ca_pins` fields to match the output from
-`tctl bots add ...`.
+Outputs must be configured with a destination. In this example, the `directory`
+destination will be used. This will write artifacts to a specified directory on
+disk. Ensure that this directory can be written to by the Linux user that
+`tbot` runs as, and that it can be read by the Linux user that will be accessing
+applications.
 
-Machine ID also allows you to use Linux ACLs to control access to certificates
-on disk. You will use this to ensure only your application has access to the
-short-lived certificates Machine ID uses.
+Modify your `tbot` configuration to add a `database` output:
 
-We'll work with the assumption you will be running Machine ID as the Linux user
-`teleport` and your application as the Linux user `app`. Create and initialize
-the output destination by running this `tbot init` command either as `root`
-or as the `teleport` user:
-
-```code
-$ tbot init \
-    -c /etc/tbot.yaml \
-    --init-dir=/opt/machine-id \
-    --bot-user=teleport \
-    --owner=teleport:teleport \
-    --reader-user=app
+```yaml
+outputs:
+- type: database
+  destination:
+    type: directory
+    path: /opt/machine-id
+  # Specify the details of the database you wish to connect to.
+  service: example-server
+  database: example
+  username: alice
+  # Specify a format to use for the output credentials. For most databases,
+  # this configuration field can be omitted. See the table below for the
+  # databases that require a value to be specified here.
```

Comment:
```suggestion
  # Specify a format to use for the output credentials. For most databases,
  # this configuration field can be omitted.
```
Since the table is now above the example YAML